llvm/test/tools/llvm-tli-checker/ps4-tli-check.yaml

```diff
@@ -263,6 +263,10 @@ DynamicSymbols:
     Type:            STT_FUNC
     Section:         .text
     Binding:         STB_GLOBAL
+  - Name:            atexit
```

Comment:
@pogo59 @sks75220 Let me know for ps4/ps5if you prefer that this library function be set unavailable.

Reply:
`atexit` should be available for PS4/PS5, thanks!